packages/neuron-ui/src/components/ApproveMultisigTxDialog/index.tsx

```diff
@@ -152,12 +149,12 @@ const ApproveMultisigTxDialog = ({
             <div className={styles.conciseData}>
               <div className={styles.inputWrap}>
                 <h2>Inputs</h2>
-                {offlineSignJson.transaction?.inputs?.map((input: CellProps) => (
-                  <Cell cell={input} isMainnet={isMainnet} key={input.lockHash} />
-                ))}
+                {offlineSignJson.transaction?.inputs?.map(input =>
+                  input.lock ? <Cell cell={input} isMainnet={isMainnet} key={input.lockHash} /> : null
```

Comment:
Why is it not displayed when there is no `input.lock`? Is this a business logic?